src/implementations/vanilla/components/page-header/page-header-ec.scss

```diff
@@ -138,14 +138,15 @@
  * background image variant
  */
 .ecl-page-header--image {
-  position: relative;
+  .ecl-page-header__background-container {
+    position: relative;
+  }
 
   .ecl-page-header__background {
-    background-position: 50% 0;
-    background-size: cover;
     display: block;
     height: 10.875rem;
-    position: relative;
+    object-fit: cover;
+    object-position: 50% 0;
```

Comment:
mmh..i understand we want to reproduce the current "look and feel" with the image used in the demo, but this property seems only meaningful in this specific use case, while there is no clear "requirement" or guildeline about the image to be uploaded and the portion of it that will be shown by the browser.

Reply:
If I remember well, that was requested at some point to center the image vertically. 
Maybe in the future we could offer a paramter to change the image position, but for now I think that the safest move is to keep what we have currently (visually). Otherwise that may considered as a breaking change